gbpservice/nfp/base_configurator/api/v1/controllers/controller.py

```diff
@@ -111,14 +201,25 @@ def post(self, **body):
             config_data = body['config'][0]
             context = config_data['kwargs']['context']
             request_info = config_data['kwargs']['request_info']
+            if 'device_ip' in request_info:
+                msg = ("PUTTING DATA TO VM :: %s" % body)
+                LOG.info(msg)
+                device_ip = request_info['device_ip']
+                cache_ips.add(device_ip)
+                ip = str(device_ip)
+                requests.post(
+                    'http://'+ip+':8080/v1/nfp/'+self.method_name,
+                    data=jsonutils.dumps(body))
 
-            # Only heat is supported presently
-            if (service_type == "heat"):
-                result = "unhandled"
-                self._push_notification(context, request_info, result)
             else:
-                result = "error"
-                self._push_notification(context, request_info, result)
+                if (service_type == "heat"):
```

Comment:
This is retained for handling the case where device_ip is not there. But anyway this need to be changed to config_init.